File: FinBoard/FrontEnd/src/app/models/order-filters.ts
export interface OrderFilters {
  id: string | null;
  isin: string | null;
  quantity: string | null;
  portfolio: string | null;
  operationDateFrom: string | null;
  operationDateTo: string | null;
  valueDateFrom: string | null;
  valueDateTo: string | null;
  status: string | null;
  currency: string | null;
  side: string | null;
}

File: FinBoard/BackEnd/src/main/java/pwork/greco/antonio/finboard/dto/OrderFilters.java
package pwork.greco.antonio.finboard.dto;

import lombok.Data;

import java.math.BigDecimal;

@Data
public class OrderFilters {
    private String id;
    private String isin;
    private BigDecimal quantity;
    private Long  portfolio;
    private String operationDateFrom;
    private String operationDateTo;
    private String valueDateFrom;
    private String valueDateTo;
    private String status;
    private String currency;
    private String side;
}
